azure-pipelines.yml

```diff
@@ -51,6 +51,16 @@ steps:
   workingDirectory: $(Pipeline.Workspace)/target/python-wheels/
   displayName: 'Install Python dependencies'
 
+- script: |
+    set -ex
+    # Install .NET CORE
+    wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+    sudo dpkg -i packages-microsoft-prod.deb
```

Comment:
Why download this package? Can you not simply use `apt-add-repository` as demonstrated in the "Ubuntu 20.04 (Focal)" section here: https://docs.microsoft.com/en-us/windows-server/administration/linux-package-repository-for-microsoft-software.